test/Npgsql.Tests/SystemTransactionTests.cs

```diff
@@ -341,7 +489,7 @@ public void SchemaConnectionShouldntEnlist()
         }
 
         void AssertNoPreparedTransactions()
-            => Assert.That(GetNumberOfPreparedTransactions(), Is.EqualTo(0));
+            => Assert.That(GetNumberOfPreparedTransactions(), Is.EqualTo(0), "Prepared transactions found");
 
         int GetNumberOfPreparedTransactions()
```

Comment:
Locally I have tried using that for adding a `AssertHasPreparedTransactions` in most distributed tests, but that assert was failing everywhere, finding nothing prepared.